contracts/QuotaControlled.sol

```diff
@@ -0,0 +1,108 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./interfaces/IQuotaControlled.sol";
+
+contract QuotaControlled is IQuotaControlled {
```

Comment:
Awkward name, open to suggestions

Reply:
Naming things...hardest task in programming :)

If openzeppelin dared to call a contract that manages an owner ... Ownable, then this one that manages Quotas could be Quotable 😁

Reply:
Maybe Quota'ble :smile: 
I was thinking of AccessControl when I chose this, but that contract controls access, while this contract is controlled (limited?) by quotas (hence the 'ed).
Throttleable maybe :thinking: (at least it's a word)

Reply:
I decided on QuotaEnforcer. It's not great but I don't see it being misunderstood.